packages/ketcher-react/src/script/ui/data/convert/structConverter.ts

```diff
@@ -136,45 +95,56 @@ export function toString(struct, format, server, serverOpts) {
   })
 }
 
-export function fromString(structStr, opts, server, serverOpts) {
+export function fromString(
+  structStr: any,
```

Comment:
if here is `Str`, so should we use `string` type instead of `any`? In this case you can remove `Str` postfix

Reply:
fixed